tests/app/main/views/test_services.py

```diff
@@ -849,6 +900,7 @@ def test_cannot_make_removed_service_public(self, data_api_client):
             'supplierId': 2,
             'status': 'disabled'
         }}
+        data_api_client.find_audit_events.return_value = {'auditEvents': []}
```

Comment:
I'm sure there's a way to set this in the patch at the class level too, rather than having to add the same value  to each test - I have seen it done in our codebase somewhere. Probably by Rob. But I can't seem to remember/find the syntax right now 😞

Reply:
I think you can yield the return value in a fancy way, yes. Will have a look....

Reply:
We can do something like:

`@mock.patch('app.main.views.services.data_api_client', autospec=True, **{'find_audit_events.return_value': {'auditEvents': []}})`

It's a bit messy :(

Reply:
Yeah I wasn't saying we definitely need it.  That's not so pretty, so fine as is I reckon.